src/utils/__tests__/ImagePicker.test.ts

```diff
@@ -0,0 +1,53 @@
+import 'react-native';
+import * as ImagePicker from '../ImagePicker';
+
+jest.mock('expo-permissions', () => ({
+  ...require.requireActual('expo-permissions'),
+  askAsync: jest.fn(),
+}));
+
+/* eslint-disable import/first */
+import { askAsync } from 'expo-permissions';
+
+jest.mock('expo-image-picker', () => ({
+  launchCameraAsync: (): string => 'photo info',
+  launchImageLibraryAsync: (): string => 'photo info',
+}));
+
+describe('ImagePicker interaction', () => {
+  it('launchCameraAsync should return photo info when permission is granted', async () => {
+    // @ts-ignore
+    askAsync.mockReturnValueOnce({
+      status: 'granted',
+    });
+    const result = await ImagePicker.launchCameraAsync();
+    expect(result).toBe('photo info');
+  });
+
+  it('launchImageLibraryAsync should return photo info when permission is granted', async () => {
+    // @ts-ignore
+    askAsync.mockReturnValueOnce({
+      status: 'granted',
+    });
+    const result = await ImagePicker.launchImageLibraryAsync();
+    expect(result).toBe('photo info');
+  });
+
+  it('launchCameraAsync should not return photo info when permission is not granted', async () => {
```

Comment:
better naming would be
it('should launchCameraAsync and return photo info when permission is granted')

Reply:
thank you!